drivers/input/input_cst816s.c

```diff
@@ -161,7 +161,7 @@ static int cst816s_process(const struct device *dev)
 	LOG_DBG("gesture: %d", gesture);
 
 	if (gesture != CST816S_GESTURE_CODE_NONE) {
-		input_report(dev, INPUT_EV_MSC, (uint16_t)gesture, 0, true, K_FOREVER);
+		input_report(dev, INPUT_EV_DEVICE, (uint16_t)gesture, 0, true, K_FOREVER);
```

Comment:
nit: can you swap the order of the two patches and add the new even type first so that the second has the final value?